apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -399,6 +399,7 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
           </DialogDescription>
 
           <SignaturePad
+            containerClassName={isSubmitting ? 'pointer-events-none opacity-50' : undefined}
```

Comment:
Please create a `disabled` prop on the SignaturePad and update the logic there.

Reply:
@dguyen I've added `isDisabled` props on the signing pad and have fixed the usage at all the places. Please review.

Reply:
Can you please use `disabled` so it matches normal input elements